### DISCUSSION TITLE
Voltage for powering board and reference for other split-phase (Is 9 & 12VAC nominal or actual maximum?)

### DISCUSSION THREAD

**Discussion by alexruffell** [CircuitSetup/Expandable-6-Channel-ESP32-Energy-Meter#109] [Q&A] [answered]
The documentation I've read says to use 9VAC, and in some cases up to 12VAC to power the boards, and for the reference for the other split-phase. On the board itself it says not to exceed 12VAC.

However... transformers that say 9VAC output a few volts above that in order to not sag below 9VAC at full load. The 9V is the nominal value and the real output will be something like 11VAC (I measured this on the recommended JAMECO 9VAC transformer). The small 9VAC transformer I had actually outputs 14VAC at no load... so I am concerned this will cause damage in the long run.

The main transformer has two 6VAC secondary coils that can be wired in series to achieve 12VAC (which the documentation said was ok, but is not super clear). The voltage I measure is between 15 and 16VAC.

I am in the final stages of assembly/install so I really need to know whether this will be an issue. I do not want to damage the boards (2 once I get the add-on)! What is the hard limit before damage occurs?

The reason I ask is that 9VAC and 12VAC transformers are recommended, which as far as I know will always output more since they are unregulated. Therefore, the actual maximum before damage occurs should be high enough to tolerate some variability.

![20220814_172726](https://user-images.githubusercontent.com/4006944/184558570-f840aac2-101f-49e5-b1e0-3456e46618ce.jpg)

**Comment by CircuitSetup** [comment, marked as answer]
Yes, the transformer output voltage will be okay for the meter.  The power supply can actually take up to 24V. You'll just need to calibrate the meter to the transformer to get an accurate reading.

**Reply by alexruffell** [reply]
Awesome, thank you for confirming @CircuitSetup !

**Reply by ahasbini** [reply]
Thanks @CircuitSetup , is it alright if the README mentioned the 24VAC limit? I found it difficult to figure out on my own until I found this discussion. Thanks.
